[REVIEW] Added RSNs for Rocky transition,  CUDA 11.0 Deprecation, and new docker parameters (#274)
rapidsai/docs: _notices/rsn0019.md

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 19 # should match notice number
notice_pin: true # set to true to pin to notice page
title: "Stop supporting `CentOS 8` in v22.08"
notice_author: RAPIDS Ops
notice_status: In Progress
notice_status_color: yellow
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change
notice_rapids_version: "v22.08"
notice_created: 2022-06-08
# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2022-06-08
---

## Overview

`CentOS 8` is now at end-of-life, which means RAPIDS will no longer publish any images built on `CentOS 8` following the RAPIDS `v22.06` release.

> Additionally, gpuCI CPU builds will be converted over to use `Rocky 8.5` instead of `CentOS 8`.

Reviewer: ```suggestion
Additionally, gpuCI CPU builds will be converted over to use `Rocky 8.6` instead of `CentOS 8`.
```

## Status

> Dropping non-LTS support for `CentOS 8` will complete after RAPIDS `v22.08` release.

Reviewer: What do we mean by non-LTS support here?

## Impact

> Users should consider switching to any of the following supported operating systems for any `rapidsai/rapidsai-nightly` image:

Reviewer: ```suggestion
Users should consider switching to any of the following container OSes for any `rapidsai/rapidsai-nightly` image:
```

  - Ubuntu 18.04
  - Ubuntu 20.04

>   - Rocky 8.5

Reviewer: ```suggestion
  - Rocky 8.6
```
